### PR TITLE
Smart wallet mobile SDK 0.1.0 doc update

### PR DESCRIPTION
**Pull request by fan-zhang-sv**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: docs/pages/guides/react-native-integration.mdx [hunk 1]
```diff
@@ -21,7 +21,7 @@ Opening `https://your-app.example.com/coinbase-wallet-sdk` should open up your a
 You can see more detailed instructions on setting up deeplinks in the [Expo docs](https://docs.expo.dev/guides/deep-linking/).
 
 <Callout type="warning" >
-Note that custom scheme (e.g. `myapp://`) is currently not supported by the Smart Wallet due to security concerns.
+When instantiating SDK, `appDeeplinkUrl` is required to establish your app's identity, use 'https://' link for production. If a custom scheme `myapp://` is provided, user will see a banner warning about the insecure connection.
```
Comment (nateReiners) [inline docs/pages/guides/react-native-integration.mdx:24]: Couple of minor grammar adjustments

```suggestion
When instantiating the SDK, `appDeeplinkUrl` is required to establish your app's identity. Use 'https://' links for production. If a custom scheme `myapp://` is provided, users will see a banner warning about the insecure connection.
```

File: docs/pages/guides/react-native-integration.mdx [hunk 2]
```diff
@@ -144,12 +144,17 @@ Create a new `CoinbaseWalletSDK` instance and use the `makeWeb3Provider` functio
 
 The `appDeeplinkUrl` is required and should match the deeplink URL you set up earlier. 
 
+<Callout type="success" >
+A new config `appCustomScheme` is introduced to remove the Done screen after signing.
```
Comment (nateReiners) [inline docs/pages/guides/react-native-integration.mdx:148]:
```suggestion
The `appCustomScheme` config option can be used to remove the Done screen after signing.
```

File: docs/pages/guides/react-native-integration.mdx [hunk 3]
```diff
@@ -187,6 +192,9 @@ yarn add @mobile-wallet-protocol/wagmi-connectors@latest
 
 Simply import the `createConnectorFromWallet` function and pass in the wallet you want to use to wagmi config.
 
+<Callout type="success" >
+A new config `appCustomScheme` is introduced to remove the Done screen after signing.
+</Callout>
```
Comment (nateReiners) [inline docs/pages/guides/react-native-integration.mdx:197]: This looks the same as above. Redundant?

**Review by nateReiners — COMMENTED** [review]
Looks good, I just left a couple of minor wording changes

File: docs/pages/guides/react-native-integration.mdx (hunk 3 above)
Comment (fan-zhang-sv) [reply docs/pages/guides/react-native-integration.mdx:197]: we provide 2 interfaces here, some user may jump right into the second one, would like to emphasize in both section

**Review by nateReiners — APPROVED** [review]

**Pull request merged**